source/IntegrationEventListener/Energinet.DataHub.Aggregations.DatabaseMigration/Scripts/202203250902_Create_Charge_Table.sql

```diff
@@ -0,0 +1,18 @@
+﻿CREATE TABLE [dbo].[Charge]
+(
+	[RowId] [uniqueidentifier] NOT NULL,
+	[ChargeKey] [nvarchar](max) NOT NULL,
+	[ChargeId] [nvarchar](255) NOT NULL,
```

Comment:
These are hard to distinguish. I suggest adopting the naming from the charges domain:
```suggestion
	[Id] [nvarchar](255) NOT NULL,
	[SenderProvidedChargeId] [nvarchar](max) NOT NULL,
```

Reply:
I don't think they are hard to distinguish. In all fairness we have used this naming quite a few places in aggregations and therefor I see no reason to change it.

Reply:
Can we add documentation then? I don't what they are each.

Reply:
I believe the documentation comes when adding the .NET classes to correspond to the database tables.